inference-engine/src/transformations/src/transformations/common_optimizations/broadcast_elementwise_fusion.cpp

```diff
@@ -9,7 +9,14 @@
 
 NGRAPH_RTTI_DEFINITION(ngraph::pass::BroadcastElementwiseFusion, "BroadcastElementwiseFusion", 0);
 
-bool is_eliminate_broadcast(const ngraph::PartialShape & input_shape, const ngraph::PartialShape & broadcast_shape) {
+bool is_eliminate_broadcast(const ngraph::Output<ngraph::Node>& eltwise,
```

Comment:
Let's take this chance to rename this function. The current name is somewhat confusing.

```suggestion
bool can_eliminate_broadcast(const ngraph::Output<ngraph::Node>& eltwise,
```